### PR TITLE
Provider Release Process 

### PR DESCRIPTION
**Pull request by cgriggs01**
Hi @markpeek and @Prativa20,

This PR address the required remained changes needed for the vRA7 provider release.

- Added HashiCorp standard Terraform provider GNUmakefile + scripts
- Updated Go Module
- Updated import paths to new provider repository URL

Please review and approve this PR and we can move forward with the release. Thanks!

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,55 +1,8 @@
...
-	github.com/stretchr/testify v1.2.2 // indirect
-	github.com/ulikunitz/xz v0.5.4 // indirect
-	golang.org/x/crypto v0.0.0-20170930174604-9419663f5a44 // indirect
-	golang.org/x/net v0.0.0-20171004034648-a04bdaca5b32 // indirect
-	golang.org/x/sync v0.0.0-20181108010431-42b317875d0f // indirect
-	golang.org/x/sys v0.0.0-20170927054621-314a259e304f // indirect
-	golang.org/x/text v0.0.0-20171006144033-825fc78a2fd6 // indirect
-	google.golang.org/genproto v0.0.0-20171002232614-f676e0f3ac63 // indirect
-	google.golang.org/grpc v1.6.0 // indirect
+	github.com/vmware/terraform-provider-vra7 v0.1.0
```
Comment (markpeek) [inline go.mod]: This should be changed: s/vmware/terraform-providers/

File: go.sum [hunk 2]
```diff
@@ -66,21 +68,29 @@ github.com/mitchellh/reflectwalk v0.0.0-20170726202117-63d60e9d0dbc h1:gqYjvctjt
...
 github.com/satori/go.uuid v1.1.0/go.mod h1:dA0hQrYB0VpLJoorglMZABFdXlWrHn1NEOzdhQKdks0=
+github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d h1:zE9ykElWQ6/NYmHa3jpm/yHnI4xSofP+UP6SpjHcSeM=
 github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d/go.mod h1:OnSkiWE9lh6wB0YB77sQom3nweQdgAjqCqsofrRNTgc=
+github.com/smartystreets/goconvey v0.0.0-20181108003508-044398e4856c h1:Ho+uVpkel/udgjbwB5Lktg9BtvJSh2DT0Hi6LPSyI2w=
 github.com/smartystreets/goconvey v0.0.0-20181108003508-044398e4856c/go.mod h1:XDJAKZRPZ1CvBcN2aX5YOUTYGHki24fSF0Iv48Ibg0s=
+github.com/stretchr/testify v1.2.2 h1:bSDNvY7ZPG5RlJ8otE/7V6gMiyenm9RtJ7IUVIAoJ1w=
 github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXfy6kDkUVs=
 github.com/ulikunitz/xz v0.5.4 h1:zATC2OoZ8H1TZll3FpbX+ikwmadbO699PE06cIkm9oU=
 github.com/ulikunitz/xz v0.5.4/go.mod h1:2bypXElzHzzJZwzH67Y6wb67pO62Rzfn7BSiF4ABRW8=
+github.com/vmware/terraform-provider-vra7 v0.1.0 h1:Tlcu+O0RyS9sQI1v/ezXnySpgTzmSihTwTgbNvDsnSI=
```
Comment (markpeek) [inline go.sum]: ditto

**Review by markpeek — CHANGES_REQUESTED** [review]
Additional repo name changes are needed.

File: go.mod (hunk 1 above)
Comment (cgriggs01) [reply go.mod]: Thanks 👍

**Pull request merged**